src/ActionModal/components/ActionModal.vue

```diff
@@ -410,6 +414,7 @@ export default {
     password: null,
     sending: false,
     gasEstimate: null,
+    simulateGasEstimate: null,
```

Comment:
why both?

Reply:
to keep the simulate logic. Although now it seems to be useless. I can also delete / comment it